[HUDI-391] Rename module name from hudi-bench to hudi-test-suite

yanghua opened the pull request:
## What is the purpose of the pull request

*Rename module name from hudi-bench to hudi-test-suite*

## Brief change log

  - *Rename module name from hudi-bench to hudi-test-suite*

## Verify this pull request


This pull request is a trivial rework / code cleanup without any test coverage.


## Committer checklist

 - [ ] Has a corresponding JIRA in PR title & commit
 
 - [ ] Commit message is descriptive of the change
 
 - [ ] CI is green

 - [ ] Necessary doc changes done or have another open PR
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

yanghua commented:
@n3nash Can you review this PR?

n3nash commented:
LGTM to me, couple of things to do before merging : 

1) Can you do a search of `hudi-bench` string in the module and make sure no comments are lingering that have this.
2) The diff somehow doesn't show the module name change, I'm guessing that's just github and you've already made that ?

yanghua commented:
@n3nash 

> LGTM to me, couple of things to do before merging :
> 
> 1. Can you do a search of `hudi-bench` string in the module and make sure no comments are lingering that have this.

Yes, I have searched `hudi-bench` keyword and replaced all of them, there is nothing left.

> 2. The diff somehow doesn't show the module name change, I'm guessing that's just github and you've already made that ?

Yes. I have renamed all of them, after merging, you can not find anything if you search `bench` keyword.
